.env

```diff
@@ -0,0 +1 @@
+REACT_APP_SERVER_URL=http://localhost:8000
```

Comment:
.env to gitignore, unfortinatly i have nothing to break there, but if there was your backend url i can ddos this 4fun